src/madhuraMJun20/BankingSystem.java

```diff
@@ -0,0 +1,56 @@
+package madhuraMJun20;
+
+public class BankingSystem {
+	
+	int counterD=0;
+	int counterC=0;
+	int counterP=0;
+	static int countD=0;
+	static int countC=0;
+	static int countP=0;
+	
+	void setData(int credit, int debit, int print) {
+		
```

Comment:
Why are you not set here anything? Then what's the purpose of creating this method.

Reply:
Corrected that.